pkg/antctl/handlers/agentinfo_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	monitor "github.com/vmware-tanzu/antrea/pkg/monitor"
```

Comment:
The alias is not needed, though generated code has it.

Reply:
It seems generated code doesn’t have ConnStatus.

Reply:
Sorry, I didn't get your mean, I mean the alias "monitor" is not needed

Reply:
Sorry, my bad. I misunderstood. Alias removed.